workflow/calculation/gmsim_templates/20.4.1.2/root_defaults.yaml

```diff
@@ -1,5 +1,5 @@
 hf:
-  version: 5.4.5.3_new_gnu
+  version: 5.4.5.3
```

Comment:
@jasonmotha 
I remember you didn't like this change previously.
https://github.com/ucgmsim/slurm_gm_workflow/pull/430#discussion_r826479424

I also checked, hf binary that you state "no longer existis" is still present.

Reply:
Good catch, that should be consistent at least

Reply:
![image](https://user-images.githubusercontent.com/25143301/190270104-ac6074c2-6bca-40f8-bc2b-7b812ea1644c.png)
That HF binary is no longer on Kisti.

My comment about that earlier was more related to changing 5.4.5.3 to 5.4.5.4. If I recall correctly Viktor made those the new_gnu version functionally the same as the standard Rob Graves version.

Reply:
Sorry poor screenshot 
![image](https://user-images.githubusercontent.com/25143301/190270341-c134efa2-4ee5-4c06-a315-01d7b76374fd.png)

Reply:
@jasonmotha  As you pointed out earlier (see the screenshot), NeSI still has that binary, so backward compatibility on NeSI can matter. 

I would like to see if 5.4.5.3_new_gnu and 5.4.5.3 produce identical output on NeSI. Some reference to the nature of 5.4.5.3_new_gnu might be useful. Was there a code change or was it just built with a "new GCC" compiler (what version of GCC is *new*, by the way?).

In addition, as I raised while reviewing another PR of yours, I'd like to see more description for a PR. Took me some time and effort to follow what you were trying to achieve just by looking at the code change.
 
 
![Screenshot from 2022-09-28 09-37-30](https://user-images.githubusercontent.com/466989/192630563-810b3a38-c4e0-48d2-bac1-fd1d26829ccc.png)